react/src/components/dropdown/SprkDropdown.js

```diff
@@ -203,7 +205,12 @@ class SprkDropdown extends Component {
           <div className={classNames('sprk-c-Dropdown', additionalClasses)}>
             {heading !== '' && (
               <div className="sprk-c-Dropdown__header">
-                <h2 className="sprk-c-Dropdown__title">{heading}</h2>
+                <SprkHeading
+                  element="h2"
```

Comment:
Does this need a variant? Without it, the heading component isn't doing anything.

Reply:
Looks like this was out of sync with html & angular and needs to have the bodyTwo class added. Will get it updated.